services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+from django.db.models import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
```

Comment:
The `get_movie_session_by_id` function uses `MovieSession.objects.get(id=movie_session_id)`, which will raise a `DoesNotExist` exception if no session with the given ID is found. Consider using `get_object_or_404` if this is used in a view, or handle the exception appropriately to avoid unhandled exceptions.